deployments/kustomize/image-builds/nodejs-s2i.yaml

```diff
@@ -23,7 +23,6 @@ spec:
   failedBuildsHistoryLimit: 1
   successfulBuildsHistoryLimit: 1
   strategy:
```

Comment:
not sure why this change is here?

Reply:
because it had an extra strategy that was not supposed to be there.